dev_guide/builds/build_strategies.adoc

```diff
@@ -525,3 +525,48 @@ spec:
     file to use, relative to the source `contextDir`.
     If `contextDir` is omitted, it defaults to the root of the repository.
     If `jenkinsfilePath` is omitted, it defaults to *_Jenkinsfile_*.
+
+[[jenkins-pipeline-strategy-environment]]
+=== Environment Variables
+
+To make environment variables available to the
+xref:../../architecture/core_concepts/builds_and_image_streams.adoc#pipeline-build[Pipeline build]
+process, you can add environment variables to the `jenkinsPipelineStrategy` definition
+of the `BuildConfig`.
+
+The environment variables defined there are set as parameters on the job in Jenkins associated with
+the `BuildConfig`. 
+
+For example:
+
+[source,yaml]
+----
+jenkinsPipelineStrategy:
+...
+  env:
+    - name: "FOO"
+      value: "BAR"
+
+----
+
+Specifics on the mapping between `BuildConfig` environment variables and Jenkins job parameters:
+
+* When a Jenkins job is created or updated based on changes to a Pipeline strategy `BuildConfig`, any environment
+variables in the `BuildConfig` are mapped to Jenkins job parameters definitions, where the default values for the Jenkins job parameters 
+definitions are the current values of the associated environment variables.
+* You can still add additional parameters (whose names differ from the names of the environment variables in the `BuildConfig`) 
+to the Jenkins job from the Jenkins console after the job's initial creation and they will be honored when builds are started for those Jenkins jobs.
+* How you start builds for the Jenkins job then dictates how the parameters are set.
```

Comment:
I feel like this line and all lines below it can come under the same dot point. Or maybe something like:

* How you start Jenkins job builds dictate how the parameters are set:
** Start method 1
** Start method 2
etc